MrtCore/wpf_packaged_app/wpf_packaged_app (Package)/Package.appxmanifest

```diff
@@ -0,0 +1,49 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities"
+  IgnorableNamespaces="uap rescap">
+
+  <Identity
+    Name="25dee5b5-8e25-431f-a644-e75d98163029"
+    Publisher="CN=rohanp"

```

Comment:
nit: I wonder if maybe we might want to change this. :)

FWIW, in some of the other samples it looks like `CN=Microsoft Corporation` is used.
Ex: https://github.com/microsoft/Project-Reunion-Samples/blob/main/MrtCore/winui_desktop_packaged_app/winui_desktop_packaged_app%20(Package)/Package.appxmanifest#L11